recipes/sbp/all/conanfile.py

```diff
@@ -0,0 +1,78 @@
+import os
+
+from textwrap import dedent
+
+import yaml
+
+from conans import ConanFile, CMake, tools
+
+
+class SbpConan(ConanFile):
+    name = "sbp"
```

Comment:
I think `libsbp` is a more fitting name for this recipe. It's also the name of the `swift-nav` github repo.

Reply:
The name of the package on various package management indexes is `sbp`.
https://pypi.org/project/sbp
https://hackage.haskell.org/package/sbp
https://www.npmjs.com/package/sbp

Reply:
But they refer to themselves as libbsp: https://swift-nav.github.io/libsbp/

Reply:
follow the project name, which is `libsbp`

Reply:
"libsbp" is the repo name, so it creeps into various docs, but "SBP" is the name used for it.

fwiw, https://swift-nav.github.io/libsbp/ is an unmaintained page.  The download links refer to v2.4.7

It is now in vcpkg and spack as "sbp".  See https://repology.org/project/sbp/versions

The Rust package has now also been published, again named sbp https://lib.rs/crates/sbp

Reply:
Thanks for the links. `sbp` looks good then.